Packs/CortexDataLake/Integrations/CortexDataLake/CortexDataLake.py

```diff
@@ -192,6 +192,20 @@ def human_readable_time_from_epoch_time(epoch_time: int, utc_time: bool = False)
     return result
 
 
+def epoch_to_milliseconds_timestamp(epoch_time: int):
```

Comment:
adding the 1 mili should be separate i think, or at least change the function name

Reply:
done

Reply:
update the function name to:
epoch_to_timestamp_and_add_milli